README.md

```diff
@@ -58,6 +60,53 @@ spec:
             - Resource
 ```
 
+### Artifact Hub
+1. A dedicated folder must be created for each policy.
+2. The folder must be named the same as the policy.
+3. Use dashes for folder name and policy name instead of underscores.
+
+Add an `artifacthub-pkg.yml` metadata file to the folder. See an example metadata file for Kyverno policies below and customize per the comments.
+```yaml
+---
+name: backup-all-volumes # The name of the package (only alphanum, no spaces, dashes allowed) (required)
+version: 1.0.0 # Version of the policy (required)
+displayName: Backup All Volumes  # Display name of the policy (required)
+createdAt: "2023-03-29T00:00:00.000Z" # The date this package was created (RFC3339 layout) (required)
+description: >-
+      In order for Velero to backup volumes in a Pod using an opt-in approach, it
+      requires an annotation on the Pod called `backup.velero.io/backup-volumes` with the
+      value being a comma-separated list of the volumes mounted to that Pod. This policy
+      automatically annotates Pods (and Pod controllers) which refer to a PVC so that
+      all volumes are listed in the aforementioned annotation if a Namespace with the label
+      `velero-backup-pvc=true`.
+    # A short description of the package (required)
+digest: 60ca548c88fc3e43db880bd5e466e4fa02af13b7f97c652aee59cb13ca9404e5 
+# The SHA256 hash String that uniquely identifies this package version (optional)
+install: |-
+    ```shell
+    kubectl apply -f https://raw.githubusercontent.com/kyverno/policies/main/velero/backup-all-volumes/backup-all-volumes.yaml
+     ```
+     # The installation instructions for the package (optional)
+keywords:
+  - velero
+  - kyverno
+  # Keywords associated with the package (optional)
+readme: |
+  In order for Velero to backup volumes in a Pod using an opt-in approach, it
+  requires an annotation on the Pod called `backup.velero.io/backup-volumes` with the
+  value being a comma-separated list of the volumes mounted to that Pod. This policy
+  automatically annotates Pods (and Pod controllers) which refer to a PVC so that
+  all volumes are listed in the aforementioned annotation if a Namespace with the label
+  `velero-backup-pvc=true`.
+
+  Refer to the documentation for more details on Kyverno annotations: https://artifacthub.io/docs/topics/annotations/kyverno/
+# The README of the package (optional)
+annotations:
+  kyverno/category: "Velero"
+  kyverno/kubernetesVersion: "1.25"
+  kyverno/subject: "Pod, Annotation"
+# Annotations associated with the package (optional)
```

Comment:
Same as previous. Tell them where to get these annotations. Point them here: https://artifacthub.io/docs/topics/annotations/kyverno/